src/HTMLConverter.php

```diff
@@ -111,4 +132,40 @@ public function td(string $fieldname_attribute_name): self
 
         return $clone;
     }
+
+    /**
+     * Create a DOMElement representing a single record of data
+     *
+     * @param string $recordTagName
+     * @param string $fieldTagName
+     * @param array $record
+     * @param DOMDocument $doc
+     *
+     * @return DOMElement
+     */
+    private function createRecordRow(string $recordTagName, string $fieldTagName, array $record, DOMDocument $doc) : DOMElement
+    {
+        $node = $this->xml_converter->rootElement($recordTagName)->fieldElement($fieldTagName)->import([$record], $doc);
+
+        /** @var DOMElement $element */
+        foreach ($node->getElementsByTagName($fieldTagName) as $element) {
+            $element->setAttribute('scope', 'col');
+        }
+
+        return $node;
+    }
+
+    /**
+     * Style the table dom element
+     *
+     * @param DOMElement $table_element
+     *
+     * @return DOMElement
+     */
+    private function styleTableElement(DOMElement $table_element) : DOMElement
```

Comment:
I don't think `styleTableElement` needs to return anything the object is already known. :wink:

Reply:
changed